tests/contrib/operators/test_databricks_operator.py

```diff
@@ -165,7 +165,7 @@ def test_init_with_templating(self):
         }
         dag = DAG('test', start_date=datetime.now())
         op = DatabricksSubmitRunOperator(dag=dag, task_id=TASK_ID, json=json)
-        op.json = op.render_template('json', op.json, {'ds': DATE})
+        op.json = op.render_template(op.json, {'ds': DATE})
```

Comment:
Is it worth changing these sort of uses in the tests to `op.render_template_fields({'ds': DATE})` now?

Reply:
Yes, no point in calling `render_tempate` anymore directly because it's now more of an internal method being called by `render_template_fields`.